documentation/shared/attributes.adoc

```diff
@@ -38,11 +37,15 @@
 :kafka-exporter-project: link:https://github.com/danielqsj/kafka_exporter[Kafka Exporter^]
 
 //OAuth attributes and links
+:oauth: OAuth 2.0
 :oauth2-site: link:https://oauth.net/2/[{oauth} site^]
 :oauth-artifact-version: 0.2.0
-:oauth-blog: link:https://strimzi.io/2019/10/25/kafka-authentication-using-oauth-2.0.html[Kafka authentication using {oauth}]
-:oauth-demo-keycloak: link:https://github.com/strimzi/strimzi-kafka-oauth/tree/master/examples[Using Keycloak as the {oauth} authorization server]
-:oauth-demo-hydra: link:https://github.com/strimzi/strimzi-kafka-oauth/tree/master/examples/docker[Using Hydra as the {oauth} authorization server]
+:oauth-server: Keycloak
```

Comment:
Could you please rename the variable? When reading the docs (+ updating them in the future etc.), this suggests that it is some generic term. But it is Keycloak, so maybe it should be `:keycloak-server:`? The same for all the other links / variables which are to Keycloak. It will make it much easier to read the raw asciidoc files.

Reply:
This would be a different value downstream, for example, Single Sign-On.

Reply:
@scholzj - I would prefer we left as is, so we can use the same attribute upstream/downstream.

Reply:
So why cannot the atribute downstream be keycloak? I know it will resolve to RHSSO, but I think it will still be more readable than with `oauth`.

Reply:
what about `authorization-server`?

Reply:
I guss that is better. But it is still very generic name for a very specific feature whcih exists only in one project - Keycloak (+ RHSSO product).

Reply:
Okay. If it makes things easier, we'll go with `keycloak-server`.  We only use the other keycloak attribute (oauth-server-doc) in a couple of places, and its just a link to doc, so I'll leave it as it is.

Reply:
@scholzj - attribute name changed :+1: